app/src/main/kotlin/com/wire/android/WireApplication.kt

```diff
@@ -112,7 +112,7 @@ class WireApplication : Application(), Configuration.Provider {
                     .detectDiskReads()
                     .detectDiskWrites()
                     .penaltyLog()
-                    .penaltyDeath()
+//                    .penaltyDeath()
```

Comment:
Did this slip to this PR mistakenly? Not sure if we agreed to this in collective. Last information I got was that we should keep the penalty death on debug builds.